recipes/pyflowy/recipe.yaml

```diff
@@ -0,0 +1,88 @@
+schema_version: 1
+
+context:
+  name: pyflowy
+  version: 0.0.1
+  build_num: 0
+  shasum: babe8f58e1d2cc3784a87a50bfbb904cac088ad51259a7a2ccb835d5e3f0dfd1
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/pyflowy-${{ version }}.tar.gz
+  sha256: ${{ shasum }}
+
+build:
+  skip:
+    - if: python < 3.9
+      then: true
+    - win
+    - osx
+  number: ${{ build_num }}
+  string: h${{ hash }}_${{ build_num }}
+  script:
+    - ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - ninja
+    - meson
+    - xtensor <0.25.0
+    - xtensor-blas
+    - xtensor-python
+    - cmake
+    - ${{ stdlib('c') }}
+    - if: (build_platform != target_platform)
+      then:
+        - python
+        - cross-python_${{ target_platform }}
+  host:
+    - python
+    - meson-python
+    - pybind11
+    - pkgconfig
+    - pip
+    - fmt
+    - numpy
+    - libnetcdf
+    - hdf5
+    - ${{ compiler('c') }}
```

Comment:
`compiler(...)` should go up in the `build` section. I believe `host` and `build` should be reorganized. There's several things that should be in the `build_platform != target_platform` section. The following recipe might be a good guide:
https://github.com/conda-forge/matplotlib-feedstock/blob/5da242b3aa49613299e2bc176011d22aac266bf3/recipe/meta.yaml